go/vt/vttablet/tabletmanager/restore.go

```diff
@@ -44,6 +51,13 @@ var (
 	restoreFromBackup     = flag.Bool("restore_from_backup", false, "(init restore parameter) will check BackupStorage for a recent backup at startup and start there")
 	restoreConcurrency    = flag.Int("restore_concurrency", 4, "(init restore parameter) how many concurrent files to restore at once")
 	waitForBackupInterval = flag.Duration("wait_for_backup_interval", 0, "(init restore parameter) if this is greater than 0, instead of starting up empty when no backups are found, keep checking at this interval for a backup to appear")
+
+	// Flags for PITR
+	binlogHost           = flag.String("binlog_host", "", "(PITR restore parameter) host name of binlog server.")
+	binlogPort           = flag.Int("binlog_port", 0, "(PITR restore parameter) port of binlog server.")
+	binlogUser           = flag.String("binlog_user", "", "(PITR restore parameter) username of binlog server.")
+	binlogPwd            = flag.String("binlog_password", "", "(PITR restore parameter) password of binlog server.")
+	timeoutForGTIDLookup = flag.Duration("binlog_lookup_timeout", 60*time.Second, "(PITR restore parameter) timeout for fetching gtid from timestamp.")
```

Comment:
what do you think of `gtid_lookup_timeout` for this flag name?

Reply:
that does not signifies which `gtid` we are referring to, let me know if we should use that.

Reply:
recovery_gtid_lookup_timeout?